### PR TITLE
Texturecube modify: add test case for lua project

### PR DESCRIPTION
**Pull request by zhukaixy**

### CODE REVIEW

File: cocos/scripting/lua-bindings/auto/lua_cocos2dx_3d_auto.cpp [hunk 1]
```diff
@@ -1293,14 +1293,14 @@ int lua_cocos2dx_3d_Sprite3D_createAsync(lua_State* tolua_S)
...
-		;
+            // Lambda binding for lua is not supported.
+            assert(false);
+        } while(0)
+        ;
             if (!ok) { break; }
             void* arg3;
             #pragma warning NO CONVERSION TO NATIVE FOR void*
-		ok = false;
+        ok = false;
```
Comment (super626) [inline cocos/scripting/lua-bindings/auto/lua_cocos2dx_3d_auto.cpp]: Check indent

**Comment by zhukaixy** [comment]
Texturecube Modify

File: cocos/3d/CCSkybox.cpp [hunk 2]
```diff
@@ -97,16 +89,24 @@ void Skybox::initBuffers()
...
-        7, 6, 2, 7, 2, 3, // left
-        2, 6, 5, 2, 5, 1, // up
-        3, 0, 4, 3, 4, 7  // down
-    };
+                                1, 5, 4, 1, 4, 0, // right
+                                4, 5, 6, 4, 6, 7, // back
+                                7, 6, 2, 7, 2, 3, // left
+                                2, 6, 5, 2, 5, 1, // up
+                                3, 0, 4, 3, 4, 7  // down
+                             };
```
Comment (super626) [inline cocos/3d/CCSkybox.cpp]: wrong indent

File: tests/cpp-tests/Classes/Sprite3DTest/Sprite3DTest.cpp [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
- 
- THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- THE SOFTWARE.
+    Copyright (c) 2012 cocos2d-x.org
+    Copyright (c) 2013-2014 Chukong Technologies Inc.
```
Comment (super626) [inline tests/cpp-tests/Classes/Sprite3DTest/Sprite3DTest.cpp]: So many space

File: cocos/3d/CCSkybox.cpp (hunk 2 above)
Comment (masterhou) [inline cocos/3d/CCSkybox.cpp]: need const

**Pull request merged**